src/components/Editor/SearchBar.js

```diff
@@ -136,13 +136,13 @@ class SearchBar extends Component<Props, State> {
 
   closeSearch = (e: SyntheticEvent<HTMLElement>) => {
     const { editor, searchOn } = this.props;
-
     if (editor && searchOn) {
       this.clearSearch();
       this.props.closeFileSearch(editor);
       e.stopPropagation();
       e.preventDefault();
     }
+    this.setState({ query: "" });
```

Comment:
This is required so that the search input value is cleared upon close (i.e. Escape key)

Reply:
:+1: